server/project/test/calendar_test.py

```diff
@@ -0,0 +1,164 @@
+import datetime
+from datetime import datetime as dt
```

Comment:
Could be replaced by `import datetime as dt`

Reply:
dt is actually datetime.datetime

so I could use: ` dt.utcnow()` instead of as your mention: `dt.datetime.utc.now()`